packages/form/src/assets/css/password.css

```
@@ -0,0 +1,11 @@
+.eye-icon {
```

should be scoped for password input
```suggestion
.input-field-password .eye-icon {
```